pyads/pyads_ex.py

```diff
@@ -285,6 +299,36 @@ def adsAddRouteToPLC(
     raise RuntimeError("Unexpected response from PLC")
 
 
+def adsGetNetIdForPLC(ip_address: str) -> Optional[str]:
```

Comment:
Is there a way this function returns None? I see it like this: It either returns the net id or raises an Exception. So the return type should be non-optional